app/build.gradle

```diff
@@ -26,5 +26,5 @@ android {
 
 dependencies {
     implementation 'androidx.appcompat:appcompat:1.1.0'
-    implementation 'com.gitlab.bluedotio.android:point_sdk_android:15.3.0'
+    implementation 'com.gitlab.bluedotio.android:point_sdk_android:15.4.2'
```

Comment:
To avoid having to come back and update sdk version on every release, how about we make it dynamic for all minor/patch releases by having it as `:15.+`?